.circleci/config.yml

```diff
@@ -241,6 +253,7 @@ jobs:
     steps:
       - darkcheckout
       - setup-app
+      - attach_workspace: { at: "." }
       - run:
           name: Setup backend build environment
           command: |
```

Comment:
I'm not sure what you mean here. Are you referring to the attach_workspace step?

or maybe you meant to highlight this:
```
if [[ -f /home/dark/tree-sitter.so && ! -f /home/dark/app/backend/src/LibTreeSitter/tree-sitter.so ]]; then
              mv /home/dark/tree-sitter.so /home/dark/app/backend/src/LibTreeSitter/
              else
              echo "No action needed."
            fi
```

which should be moved to the build-parser job

Reply:
This is what I meant to highlight sorry 
```
    command: |
      set -x
      if [[ -f /home/dark/tree-sitter.so && ! -f /home/dark/app/backend/src/LibTreeSitter/tree-sitter.so ]]; then
        mv /home/dark/tree-sitter.so /home/dark/app/backend/src/LibTreeSitter/
        else
        echo "No action needed."
      fi
```


In the build-backend job, we have the following part which isn't common between the build-backed and build-cli jobs
 ```
     command: |
      set -x
      scripts/devcontainer/_start-postgres
      # Disable for now since they're not used
      # scripts/devcontainer/_start-yugabyte
      if ...
 ```
 
 Should I just keep it in the build-backend job and remove the rest (i.e  `if [[ -f /home/dark/tree-sitter.so && ! -f /home/dark/app/backend/src/LibTreeSitter/tree-sitter.so ]]; then....`)

Reply:
I'll make a commit in a moment if the explanation isn't clear

Reply:
I _think_ the answer to the question is yes, based on my understanding